driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/fork.bpf.c

```diff
@@ -149,7 +149,7 @@ int BPF_PROG(fork_x,
 	auxmap__store_u32_param(auxmap, vm_swap);
 
 	/* Parameter 14: comm (type: PT_CHARBUF) */
-	auxmap__store_charbuf_param(auxmap, (unsigned long)task->comm, KERNEL);
+	auxmap__store_charbuf_param(auxmap, (unsigned long)task->comm, MAX_PROC_EXE, KERNEL);
```

Comment:
Hi @Andreagit97 ,

just looked into the kernel sources, the `comm` is a 16-byte char arrary and the kernel itself uses a `#define TASK_COMM_LEN 16` for access. I think we need to use the same define here as well.
(Also affects all the other places, where `comm` is accessed).

Reply:
great catch! thank you